src/main/java/org/fundacion/jala/converter/core/javacompiler/JavaCompiler.java

```diff
@@ -62,4 +68,12 @@ public String javaCompiler(final JavaParameter newJavaParameter) {
             }
         }
     }
+
+    /**
+     * Returns the result object for the operation.
+     * @return compilerResult
```

Comment:
here there should be a space between the lines 73 and 74

Reply:
Line removed.